### PR TITLE
fix(TBD-5710): MaprDB components can't generate code

### PR DESCRIPTION
**Pull request by kcoepeau**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

MapR components try to use custom timestamp feature (by calling HBase component code) without having to UI to do so.

**What is the new behavior?**

Add custom timestamp UI to MapR components.
Prevent any component from using HBase component code in absence of custom timestamp UI.

### CODE REVIEW

**Comment by talend-arobu** [comment]
There are some copy/paste code here. I let the other devs confirm if we can accept it.

File: main/plugins/org.talend.designer.components.bigdata/components/tHBaseInput/tHBaseInput_begin.javajet [hunk 1]
```diff
@@ -185,9 +185,15 @@ List<IMetadataTable> metadatas = node.getMetadataList();
 <%
 if ((metadatas!=null) && (metadatas.size() > 0)) {
     IMetadataTable metadata = metadatas.get(0);
-    boolean retrieveTimestamp = ElementParameterParser.getBooleanValue(node, "__RETRIEVE_TIMESTAMP__");
-    String sourceTimestampColumn = ElementParameterParser.getValue(node, "__TIMESTAMP_SOURCE_COLUMN__");
-	String timestampColumn = ElementParameterParser.getValue(node, "__TIMESTAMP_COLUMN__");
+    boolean retrieveTimestamp = false;
+    String sourceTimestampColumn = "";
+	String timestampColumn = "";
+    if(node.getElementParameter("__RETRIEVE_TIMESTAMP__") != null){
```
Comment (bchen-talend) [inline main/plugins/org.talend.designer.components.bigdata/components/tHBaseInput/tHBaseInput_begin.javajet]: in ElementParameterParser.getBooleanValue, it throw exception when the parameter is not exists, which help to show a potential bug early, as the javajet is hard to debug and code. 
As you already add these parameters in tMapRDB components xml, I'd like to revert this change in tHBaseInput/Output javajet

File: main/plugins/org.talend.designer.components.bigdata/components/tMapRDBInput/tMapRDBInput_java.xml [hunk 2]
```diff
@@ -263,6 +263,16 @@
             SHOW_IF="((isShow[SET_HADOOP_LOGIN]) AND (SET_HADOOP_LOGIN=='true'))">
             <DEFAULT>"kerberos"</DEFAULT>
         </PARAMETER>
+        
+        <PARAMETER NAME="RETRIEVE_TIMESTAMP" FIELD="CHECK" NUM_ROW="60">
```
Comment (bchen-talend) [inline main/plugins/org.talend.designer.components.bigdata/components/tMapRDBInput/tMapRDBInput_java.xml]: It's good to have same feature in MapRDB components if test pass, else deactive this with SHOW="false" is acceptable I think.

**Comment by kcoepeau** [comment]
@bchen-talend Reverted HbaseInput/Output code changes
Waiting to see if the test for MapR components pass or not
